include/tvm/runtime/module.h

```diff
@@ -71,7 +70,11 @@ class Module {
    * \note Cyclic dependency is not allowed among modules,
    *  An error will be thrown when cyclic dependency is detected.
    */
-  TVM_DLL void Import(Module other);
```

Comment:
Do we intentionally remove the TVM_DLL here?

Reply:
The DLL function was moved to the Node class